src/App.tsx

```diff
@@ -1,35 +1,28 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from 'react';
+import Counter from './components/counter/counter';
+import Header from './components/header/header';
+import './App.css';
 
 function App() {
-  const [count, setCount] = useState(0)
+  let genreList = ['Horror', 'Romantic', 'Thrilling', 'Comedy'];
+  const [selectedGenre, setSelectedGenre] = useState('Horror');
 
+  const onSelectGenre = (genre: string) => () => {
```

Comment:
'genre' is not just a string. Its one of the genres list values. Need to define ts type for them (union type), and use over the application